docs/docs/gatsby-starters.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Gatsby Starters'
+---
+
+GatsbyJS starters are premade projects packed with features which can be used for accelerating development speed. They are also a great source for learning about implementations of different features.
```

Comment:
Let's just use my old intro from the v0-README

https://github.com/gatsbyjs/gatsby/blob/master/v0-README.md#gatsby-starters